### DISCUSSION TITLE
mviewer day 2022

### DISCUSSION THREAD

**Discussion by smevel** [mviewer/mviewer#610] [General]
Date à bloquer dans votre agenda : 25 novembre 2022 à Paris :)

**Comment by lecault** [comment]
Nous avons le plaisir de vous annoncer que la journée mviewer se déroulera le 25 novembre prochain à Paris.
Inscriptions ici : https://www.eventbrite.fr/e/billets-les-deuxiemes-rencontres-mviewer-372724227197

Plus de détail, de communication dans les semaines à venir.

Le PSC mviewer.
